src/temporal/notifications/index.ts

```diff
@@ -0,0 +1,20 @@
+import { NativeConnection, Worker } from '@temporalio/worker';
+import { createActivities } from './activities';
+import createOrGetConnection from '../../db';
+import { WorkflowQueue } from '../common';
+import { TEMPORAL_ADDRESS } from '../config';
+
+export async function run() {
+  const connection = await NativeConnection.connect({
```

Comment:
Why don't we use the client function here?

Reply:
This one is for the worker, they use different libraries/APIs for connection.

Reply:
maybe also abstract this one so only one connection is used? Same as we did for client.

Reply:
Yeah, sure, we can do that.